artifactory/cli.go

```diff
@@ -2846,6 +2850,7 @@ func createDefaultUploadSpec(c *cli.Context) (*spec.SpecFiles, error) {
 		Props(c.String("props")).
 		TargetProps(c.String("target-props")).
 		Build(c.String("build")).
+		Project(c.String("project")).
```

Comment:
I think that both the "project" and "build" file spec properties shouldn't be included as part of the upload spec.
This is because both are used for filtering out files on Artifactory's side, and this isn't relevant for the upload command.